tidb-cloud/backup-and-restore-serverless.md

```diff
@@ -60,30 +97,50 @@ TiDB Cloud Serverless only supports in-place restoration. To restore your TiDB C
 3. In **Restore Mode**, you can choose to restore from a specific backup or any point in time.
 
     <SimpleTab>
-    <div label="Basic Snapshot Restore">
+    <div label="Snapshot Restore">
 
     To restore from a selected backup snapshot, take the following steps:
 
-    1. Click **Basic Snapshot Restore**.
+    1. Click **Snapshot Restore**.
     2. Select the backup snapshot you want to restore to.
 
     </div>
     <div label="Point-in-Time Restore">
 
-    This feature lets you restore a cluster to a specific state from any time within the last 90 days.
-
-    > **Note:**
-    >
-    > The **Point-in-Time Restore** feature is currently in beta.
-
-    To restore from a specific point in time, take the following steps:
+    To restore to a specific point in time, take the following steps:
 
     1. Click **Point-in-Time Restore**.
     2. Select the date and time you want to restore to.
 
     </div>
     </SimpleTab>
 
-4. Click **Restore** to begin the restoration process.
+4. In **Destination**, you can choose to restore in-place or restore to a new cluster.
+
+    <SimpleTab>
+    <div label="Restore to a New Cluster">
+
+    To restore a new cluster, take the following steps:
+
+    1. Click **Restore to a New Cluster**.
+    2. Enter a name for the new cluster.
+    3. Choose the cluster plan for the new cluster.
+    4. Set spending limits when your choice is a scalable cluster.
+    5. Set the advanced settings as you need when your choice is a scalable cluster.
+
+    </div>
+    <div label="Restore to a New Cluster">
+
+    To restore in-place, just click **In-place Restore**.
+
+   </div>
+   </SimpleTab>
+
+5. Click **Restore** to begin the restoration process.
 
-   After initiating the restore process, the cluster status changes to **Restoring**. The cluster will be unavailable during the restore process and existing connections will be terminated. Once the restore process completes successfully, you can access the cluster as usual.
+After initiating the restore process, the cluster status changes to **Restoring**. The cluster will be unavailable until the restore is complete and cluster status changes to **Available**.
+
+## Limitations
+
+- If any TiFlash replica is enabled, the replica will be unavailable for a while after the restore because data needs to be rebuilt in TiFlash.
```

Comment:
```suggestion
- If a TiFlash replica is enabled, it will be unavailable for a period after the restore, because data needs to be rebuilt in TiFlash.
```